submissions/lolaOluwadahunsi/html/ecommerce-website.html

```diff
@@ -34,8 +34,13 @@ <h2><strong>ABOUT LAPTOPS R US</strong></h2>
         </p>
     </article>
 
+<<<<<<< HEAD:submissions/lolaOluwadahunsi/html/Ecommerce website.html
+    <img src="images/header%20image%201.jpg" width="1200" height="600" alt="Main Header" content>
```

Comment:
You need to remove the attributes `width` and `height`. Also, update the name of your images that is not how they were saved.